webapp/src/components/ActivityPage/ActivityPage.js

```diff
@@ -25,17 +27,41 @@ export default class ActivityPage extends React.PureComponent {
     return <Loader active size="huge" />
   }
 
+  hasAnyPermissionOn() {
+    const {
+      approvedBalance,
+      isLandAuthorized,
+      isMortgageApprovedForMana,
+      isMortgageApprovedForRCN
+    } = this.props.wallet
+
+    return (
+      approvedBalance ||
+      isLandAuthorized ||
+      isMortgageApprovedForMana ||
+      isMortgageApprovedForRCN
+    )
+  }
+
   renderEmpty() {
     return (
       <div className="empty">
         <p>
           {t('activity.no_activity')}
           <br />
-          {t_html('activity.start', {
-            settings_link: (
-              <Link to={locations.settings}>{t('global.settings')}</Link>
-            )
-          })}
+          {this.hasAnyPermissionOn()
```

Comment:
**Question**:
The wording seems a bit confusing to me, technically you can't _trade_ on the marketplace if you don't have `approvedBalance` and `isLandAuthorized`, but even worse if you only set mortgage permissions.

I'm not sure about this, I'd like @abarmat / @eordano to take a look, it's a minor observation

Reply:
Good observation! I just changed to `hasTradingPermissions` so it only suggest you to move to marketplace if you are able to operate, anyway we should reanalyze how we should behave in case that someone approve mortgage and not the others as @NicoSantangelo says.

Reply:
I think in a way you are allowed to trade once you have both the MANA and LAND permissions regardless of Mortgages, etc.

Reply:
Superb